tests/unit/lib/test_signal_schema.py

```diff
@@ -356,6 +357,24 @@ def test_slice_nested():
     assert list(sliced.values.items()) == [("feature.aa", int)]
 
 
+def test_mutate_rename():
+    schema = SignalSchema({"name": str})
+    schema = schema.mutate({"new_name": Column("name")})
+    assert schema.values == {"new_name": str}
+
+
+def test_mutate_new_signal():
+    schema = SignalSchema({"name": str})
+    schema = schema.mutate({"age": Column("age", Float)})
+    assert schema.values == {"name": str, "age": float}
+
+
+def test_mutate_change_type():
+    schema = SignalSchema({"name": str, "age": float, "f": File})
+    schema = schema.mutate({"age": int, "f": TextFile})
```

Comment:
It should be like `mutate(age=Column("age", int), f=Column("f", TextFile))`

It will require introducing our constructor for Column that does the type conversion to sqlalchemy types. Like:
```
class Column(sa.Column):
   def __init__(self, name, type):
      super().__init__(self, name, python_to_sql(type))
```

Reply:
If that's not possible, we can consider introducing factory methods like Column.create("name", int)

Reply:
Make sense!

Reply:
The problem here is that we cannot just put `TextFile` as type of `datachain.Column` (SQLAlchemy complains about it), and also we cannot convert it to something SQL-like. 
Also, it seems little bit strange to put it into `Column` which is clearly 1-1 with DB columns (I think users will have this intuition as well)